### PR TITLE
improving_docs

### PR DESCRIPTION
**Pull request by jposada202020**
As title states

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -56,13 +56,12 @@ Usage Example
...
 
-	from busio import I2C
 	import adafruit_bme680
 	import time
 	import board
 
-	# Create library object using our Bus I2C port
-	i2c = I2C(board.SCL, board.SDA)
+	# Create sensor object, communicating over the board's default I2C bus
+    i2c = board.I2C()   # uses board.SCL and board.SDA
```
Comment (ladyada) [inline README.rst:64]: needs another tab?

**Review by ladyada — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (jposada202020) [reply README.rst:64]: You are right, thanks...

Comment (jposada202020) [reply README.rst:64]: it renders ok in my pc, strange
image

**Comment by jposada202020** [comment]
@ladyada I reformated the whole example and verified with the docs built locally. Changes in https://github.com/adafruit/Adafruit_CircuitPython_BME680/pull/41/commits/b0b6e7d3d1c4cf54db2c63cba2e4b69b1114d1a8
image

**Review by ladyada — APPROVED** [review]

**Pull request merged**